runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -247,6 +247,14 @@ public void onConfigChanged(ConfigChanged event)
 			}
 		}
 
+		if (event.getKey().equals("isResizable"))
+		{
+			Dimension size = client.getSize();
+			this.setResizable(Boolean.valueOf(event.getNewValue()));
+			client.setSize(size);
+			client.getParent().setSize(size);
```

Comment:
Do this inside a call to ``SwingUtilities.invokeLater`` otherwise we might get some issues eventually